src/commcare_cloud/python_migration_utils.py

```diff
@@ -5,6 +5,19 @@
 import six
 
 
+def open_python_dependent(path):
```

Comment:
open to other names

Reply:
I think the name should mention or imply "write" mode. Maybe `open_for_write`? The docstring can explain how it's behavior depends on the version of Python.

Reply:
Yeah and the method already lives in a module named `python_migration_utils` so I agree it doesn't need to be in the method name. `open_for_write` sounds good to me.